cmd/orb.go

```diff
@@ -567,15 +611,30 @@ func createOrb(opts orbOptions) error {
 		return err
 	}
 
-	_, err = api.CreateOrb(opts.cl, namespace, orb)
+	if !opts.noPrompt {
+		fmt.Printf(`You are creating an orb called "%s/%s".
 
-	if err != nil {
-		return err
+You will not be able to change the name of this orb.
+
+If you change your mind about the name, you will have to create a new orb with the new name.
+
+`, namespace, orb)
+	}
+
+	confirm := fmt.Sprintf("Please confirm the name of this orb: `%s/%s`.", namespace, orb)
```

Comment:
<img width="747" alt="screen shot 2019-01-08 at 13 56 39" src="https://user-images.githubusercontent.com/555144/50810561-a0a3a700-134d-11e9-87e8-160df980fa50.png">
<img width="572" alt="screen shot 2019-01-08 at 13 59 49" src="https://user-images.githubusercontent.com/555144/50810596-b7e29480-134d-11e9-9dea-7a92020eb79b.png">

I'm not a fan of those `.?` and `.:` - I'd remove the `.`, maybe? Also, double `:` in the second screenshot seems a bit weird - maybe always keeping a question mark at the end would work, too?